src/components-examples/angular/toggle/toggle-triple/toggle-triple-example.html

```diff
@@ -0,0 +1,21 @@
+<p>Three options is only allowed in lean design.</p>
```

Comment:
according to digital.sbb.ch I'd suggest to remove all subtitles and details from this examples. With doing that it represents better the lean case.